src/test/routes/style-guide/ut-iac-judicial-review-daily-hearing-list.test.ts

```diff
@@ -76,3 +76,18 @@ describe('UTIAC (JR) - Cardiff Daily Hearing List Page', () => {
         });
     });
 });
+
+describe('UTIAC (JR) - Leeds Daily Hearing List Page', () => {
+    const metaData = JSON.parse(rawMetaData)[0];
+    metaData.listType = 'UT_IAC_JR_LEEDS_DAILY_HEARING_LIST';
+
+    getPublicationMetadataStub.withArgs('xyz').resolves(metaData);
```

Comment:
I think that this test will be failed because we are using xyz for Cardiff list. Use some other random characters.